internal/cmd/local/command_kafka_start.go

```diff
@@ -0,0 +1,161 @@
+package local
+
+import (
+	"context"
+	"fmt"
+	"runtime"
+	"strconv"
+
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/log"
+	"github.com/confluentinc/cli/internal/pkg/output"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/strslice"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+	specsv1 "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/phayes/freeport"
+	"github.com/spf13/cobra"
+)
+
+func (c *kafkaCommand) newStartCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "start",
+		Short: "Start local kafka service",
+		Args:  cobra.NoArgs,
+		RunE:  c.start,
+	}
+
+	return cmd
+}
+
+func (c *kafkaCommand) start(cmd *cobra.Command, args []string) error {
+	dockerClient, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		return err
+	}
+	defer dockerClient.Close()
+
+	_, err = dockerClient.Info(context.Background())
+	if err != nil {
+		return err
+	}
+
+	// pull the image from ecr. it will be public so no creds needed?
+	// out, err := cli.ImagePull(ctx, imageName, types.ImagePullOptions{})
+	// if err != nil {
+	// 	return err
+	// }
+	// defer out.Close()
+	// io.Copy(os.Stdout, out)
+	// log.CliLogger.Tracef("Pull confluent-local image success")
```

Comment:
```suggestion
```

Reply:
this will be added back when image becomes public, to pull the image first.